AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/InitialSync/ModelSyncedEventEmitterTests.swift

```diff
@@ -86,7 +142,14 @@ class ModelSyncedEventEmitterTests: XCTestCase {
             XCTFail("Should not have completed")
         } receiveValue: { value in
             switch value {
-            case .modelSyncedEvent:
+            case .modelSyncedEvent(let modelSyncedEvent):
+                let expectedModelSyncedEventPayload = ModelSyncedEvent(modelName: "Post",
+                                                                       isFullSync: true,
+                                                                       isDeltaSync: false,
+                                                                       added: 2,
```

Comment:
should the number of added mutations be 5? as number of applied mutations are 5 at line 168-172?

Reply:
the reason why it's 2 is because the initial sync only synced 2 models, so models that were synced were 2. the mutation events coming from he reconciliation whether it's a applied or dropped) from 3rd and onwards are all due to subscription events- unrelated to the two that were synced from the initial sync